src/renderer/api/cache-db.ts

```diff
@@ -44,8 +44,6 @@ export class CacheDbAPI extends Kysely<CacheDatabase> {
     }
 
     public async init() {
-        await this.serializePlugin.setSchema(this);
```

Comment:
makes no sense to do this before the schema even exists. That works fine unless the db is in creation